content/en/monitors/configuration/_index.md

```diff
@@ -157,30 +157,17 @@ In this case, you should enable notifications for missing data. The sections bel
 
 **Note**: The monitor must be able to evaluate data before alerting on missing data. For example, if you create a monitor for `service:abc` and data from that `service` is not reporting, the monitor does not send alerts.
 
-There are two ways to deal with missing data:
-- Metric-based monitors using the limited `Notify no data` option
-- The `On missing data` option is supported by APM Trace Analytics, Audit Logs, CI Pipelines, Error Tracking, Events, Logs, and RUM monitors
 
 {{< tabs >}}
 {{% tab "Metric-based monitors" %}}
 
-`Do not notify` if data is missing or `Notify` if data is missing for more than `N` minutes.
+If you are monitoring a metric over an auto-scaling group of hosts that stops and starts automatically, notifying for `no data` produces a lot of notifications. In this case, you should not enable notifications for missing data. This option does not work unless it is enabled at a time when data has been reporting for a long period.
 
-You are notified if data is missing or if data is not missing. The notification occurs when no data was received during the configured time window.
+| Option                                                     | Description                                                                                                                                        | Notes        |
+| ---------------------------------------------------------  | -------------------------------------------------------------------------------------------------------------------------------------------------- | ------------ |
+| **Do not notify** if data is missing                       | No notification is sent if data is missing                                                                                                         | <u>Simple Alert</u>: the monitor skips evaluations and stays green until data returns that would change the status from OK <br> <u>Multi Alert</u>: if a group does not report data, the monitor skips evaluations and eventually drops the group. During this period, the bar in the results page stays green. When there is data and groups start reporting again, the green bar shows an OK status and backfills to make it look like there was no interruption.|
+| **Notify** if data is missing for more than **N** minutes. | You are notified if data is missing or if data is not missing. The notification occurs when no data was received during the configured time window.| Datadog recommends that you set the missing data window to at least two times the evaluation period. |
```

Comment:
```suggestion
| **Notify** If data is missing for more than **N** minutes. | You are notified if data is missing or if data is not missing. The notification is sent when no data was received during the configured time window.| Datadog recommends that you set the missing data window to at least two times the evaluation period. |
```

Reply:
The part about being notified "if data is missing or if data is not missing" is a little confusing. Why would you get a notification that data is not missing?